tests/pipelines/test_pipelines_summarization.py

```diff
@@ -64,8 +65,15 @@ def run_pipeline_test(self, summarizer, _):
         if model.config.__class__.__name__ not in model_can_handle_longer_seq:
             # Switch Transformers, LED, T5, LongT5 can handle it.
             # Too long.
-            with self.assertRaises(Exception):
-                outputs = summarizer("This " * 1000)
+            # For TF models, if the weights are initialized in GPU context, we won't get expected index error from
+            # the embedding layer.
```

Comment:
This is necessary if we want to use tiny models from the Hub for pipeline tests.